scripts/superdesk-authoring/authoring.js

```diff
@@ -102,7 +102,7 @@
     }
 
     function stripHtml(item) {
-        var fields = ['headline', 'abstract'];
```

Comment:
It looks like it needs to be stripped from all outputs except ninjs when published. I created a ticket for this and assigned to our sprint: [SD-4247](https://dev.sourcefabric.org/browse/SD-4247)